homeassistant/components/esphome/binary_sensor.py

```diff
@@ -26,10 +26,6 @@ async def async_setup_entry(
     )
 
 
-# Pylint gets confused with the EsphomeEntity generics -> let mypy handle member checking
-# pylint: disable=no-member
```

Comment:
@cdce8p How did you found all these? Manually or some tool?

Reply:
You can enable the `useless-suppression` check for that.
https://github.com/home-assistant/core/blob/e541bcd54d050bd5acf63137786cc1ec70fa6520/pyproject.toml#L94-L97

Reply:
Nice to know